dist.ini

```diff
@@ -86,3 +90,6 @@ critic_config = xt/release/perlcritic.rc
 [PodSyntaxTests]
 
 [Test::Kwalitee]
+[Test::EOL]
+[Test::Portability]
```

Comment:
checks your distribution for portability problems. like filenames with multiple dots and things. just a nice to have, although maybe handy if youre looking to support windows.